Tools/autotest/autotest.py

```diff
@@ -223,6 +223,7 @@ def alarm_handler(signum, frame):
     """Handle test timeout."""
     global results, opts
     try:
+        print("Alarm handler called")
```

Comment:
Commit message says `increase time allowed for tests` but it just adds this print?

Reply:
Rebasing removed the increase-time-allowed bit.

Will fix the commit message.